libcst/codemod/commands/tests/test_rename.py

```diff
@@ -0,0 +1,385 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# pyre-strict
+
+from libcst.codemod import CodemodTest
+from libcst.codemod.commands.rename import RenameCommand
+
+
+class TestRenameCommand(CodemodTest):
+
+    ONCALL_SHORTNAME = "instagram_server_framework"
+
+    TRANSFORM = RenameCommand
+
+    def test_rename_name(self) -> None:
+
+        before = """
+            from foo import bar
+
+            def test() -> None:
+                bar(5)
+        """
+        after = """
+            from baz import qux
+
+            def test() -> None:
+                qux(5)
+        """
+
+        self.assertCodemod(before, after, old_name="foo.bar", new_name="baz.qux")
+
+    def test_rename_name_asname(self) -> None:
+
+        before = """
+            from foo import bar as bla
+
+            def test() -> None:
+                bla(5)
+        """
+        after = """
+            from baz import qux
+
+            def test() -> None:
+                qux(5)
+        """
+
+        self.assertCodemod(
+            before, after, old_name="foo.bar", new_name="baz.qux",
+        )
+
+    def test_rename_attr(self) -> None:
+
+        before = """
+            import a.b
+
+            def test() -> None:
+                a.b.c(5)
+        """
+        after = """
+            import a.b
+            import d.e
+
+            def test() -> None:
+                d.e.f(5)
+        """
+
+        self.assertCodemod(
+            before, after, old_name="a.b.c", new_name="d.e.f",
+        )
+
+    def test_rename_attr_asname(self) -> None:
+
+        before = """
+            import foo as bar
+
+            def test() -> None:
+                bar.qux(5)
+        """
+        after = """
+            import baz
+
+            def test() -> None:
+                baz.quux(5)
+        """
+
+        self.assertCodemod(
+            before, after, old_name="foo.qux", new_name="baz.quux",
+        )
+
+    def test_rename_module_import(self) -> None:
+        before = """
+            import a.b
+
+            class Foo(a.b.C):
+                pass
+        """
+        after = """
+            import c.b
+
+            class Foo(c.b.C):
+                pass
+        """
+
+        self.assertCodemod(
+            before, after, old_name="a.b", new_name="c.b",
+        )
+
+    def test_rename_module_import_from(self) -> None:
+        before = """
+            from a import b
+
+            class Foo(b.C):
+                pass
+        """
+        after = """
+            from c import b
+
+            class Foo(b.C):
+                pass
+        """
+
+        self.assertCodemod(
+            before, after, old_name="a.b", new_name="c.b",
```

Comment:
And here also we should verify the same before/after with `old_name="a", new_name="b"`